DESCRIPTION

```diff
@@ -28,7 +28,7 @@ Description: This R package is intended to standardize and simplify working with
     exported from the clinical data management system (CDMS) secuTrial. The primary
     data export the package works with is a standard non-rectangular export.
 Depends:
-    R (>= 3.0.2),
+    R (>= 3.2),
```

Comment:
someone I asked to review it said that they needed to upgrade to 3.6 to install secuTrialR...?

Reply:
I was a bit surprised too that `haven 2.2.0` indicates `R >= 3.2` (I checked on their page). Should we just set `R >= 3.6` and be done with it?

Reply:
I've just installed it on 3.4.2... so I don't know what the other thing was about... I'll go speak to him

Reply:
he doesnt know... he had problems compiling with 3.5 apparently... he had to upgrade thinking that it was secutrialr... but maybe not.
We dont actually know if it works under 3.2 right...? maybe it's better to put a more recent version...

Reply:
What would you suggest?

Reply:
I dont know... there's still a lot of 3.5 users around...

Reply:
Lets stick with 3.2 as currently indicated then. At least according to the haven 2.2.0 docu this should work fine.

Reply:
If you agree with 3.2 you can merge this.